todo.md

```diff
@@ -0,0 +1,11 @@
+# TODOs for json schema generation
```

Comment:
I don't think we want a todo in the root. Can you instead add this to a github issue?